docs/manual/system/debug-mode.en.md

```diff
@@ -0,0 +1,140 @@
+---
+title: "Debug Mode"
+description: "The debug mode is useful during development of the web application and for tracking down errors."
+weight: 30
+---
+
+
+During development of your web application the debug mode (also called _developer mode_
+or _developer environment_) can be helpful for a variety of things. When enabled, 
+the following things are in effect:
+
+* Images are always generated and not loaded from cache.
```

Comment:
This is no longer true since Contao 4.5, see https://github.com/contao/core-bundle/issues/1014

Reply:
Right, I forgot about that. See https://github.com/contao/docs/pull/133